internal/redfishwrapper/boot_device.go

```diff
@@ -76,3 +77,65 @@ func (c *Client) SystemBootDeviceSet(ctx context.Context, bootDevice string, set
 
 	return true, nil
 }
+
+// bootTargetToDevice tries to convert the redfish boot target to a bmclib supported device string.
+// if the target is unknown or unsupported, then a string of the target is returned.
+func bootTargetToDevice(target rf.BootSourceOverrideTarget) (device string) {
+	switch target {
```

Comment:
Can this switch case be moved into its own method and be reused in this method and the above `SystemBootDeviceSet` method?

Reply:
The switch statement is in it's own function, but not a method. Is that ok? I didn't see a need to make it a method since it doesn't make use of the client.

The `SystemBootDeviceSet` method above is actually doing the inverse of this switch statement (string => `rf.BootSourceOverrideTarget`), but I can write something that works for both scenarios.

Reply:
that works, thanks!